index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>PORTFOLIO: SETUP AND MOBILE VERSION SKELETON</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="container">
+        <header>
+            <div id="toolbar">
+                <nav class="left">
+                    <img src="Time.png" alt="">
+                </nav>
+                <nav class="right">
+                    <img src="Cellular.png" alt="Cellular">
+                    <img src="Wifi.png" alt="Wifi">
+                    <img src="Battery.png" alt="battery">
+                </nav>
+                <nav>
+                    <div class="toolbar1">
```

Comment:
- [ ] Please refactor the logo `div` to `a` tag. The reason is that, normally, site logos are a simple way to redirect the user to the homepage.